netlify.toml

```diff
@@ -0,0 +1,5 @@
+[build]
+  publish = ".next"
+
+[[plugins]]
```

Comment:
Is this plugin required because we're building the static contents in GitHub Actions & those contents are hosted on Netlify. AFAIK the plugin helps build SSR stuff on Netlify's CI/CD environment & nothing else.